Vagrantfile

```diff
@@ -161,26 +145,14 @@ Vagrant.configure("2") do |global_config|
         v.tags = {'Name' => 'swift'}
       end
 
-      unless vagrant_box.start_with? 'jammy' then
-        # Install libssl for Chef (https://github.com/hashicorp/vagrant/issues/10914)
-        config.vm.provision "shell",
-          inline: "sudo apt-get update -y -qq && "\
-            "export DEBIAN_FRONTEND=noninteractive && "\
-            "sudo -E apt-get -q --option \"Dpkg::Options::=--force-confold\" --assume-yes install libssl1.1"
-      end
-
       config.vm.provision :chef_solo do |chef|
-        unless vagrant_box.include? "m1" then
-          chef.product = "chef-workstation"
-        end
```

Comment:
oh cool, do we not need to change the product for the macs anymore?

Reply:
i read it backwards the first time too - it was an "unless m1" i.e. "if not m1" - i think it's because we weren't using bento/chef boxes